app/models/shipment.rb

```diff
@@ -1,4 +1,14 @@
 class Shipment < ApplicationRecord
   belongs_to :company
-  belongs_to :shipment_item
+  has_many :shipment_items
+
+  validates :destination_country, presence: true
+  validates :origin_country, presence: true
+  validates :tracking_number, presence: true, uniqueness: true
+  validates :slug, presence: true
+
+  def group_items(count_list=Hash.new(0))
```

Comment:
Apologies I accidently set the main branch as the base branch instead of the branch for task-two. I have now set the base branch to task-two so it should be a bit more clear